Example/Podfile

```diff
@@ -1,4 +1,4 @@
-platform :"ios", "9.0"
+platform :"ios", "9.3"
```

Comment:
Any reason for raising min iOS version?

Reply:
Please see https://github.com/RxSwiftCommunity/RxCoreData/blob/master/RxCoreData.podspec

->  s.ios.deployment_target = '9.3'

Error was:

Analyzing dependencies
Fetching podspec for `RxCoreData` from `../`
[!] CocoaPods could not find compatible versions for pod "RxCoreData":
  In Podfile:
    RxCoreData (from `../`)

Specs satisfying the `RxCoreData (from `../`)` dependency were found, but they required a higher minimum deployment target.